src/api.js

```diff
@@ -996,7 +1004,8 @@
   };
 
   ChatConnection.prototype._initMediaController = function () {
-    if (connect.core.mediaFactory) {
```

Comment:
Should we keep the `connect.core.mediaFactory` check here?

Reply:
No need. The new check accomplishes the same goal as this one, but is more strict (it only allows agent-side connections to init media controllers).